sim/network.go

```diff
@@ -55,12 +57,14 @@ type Network struct {
 }
 
 func NewNetwork(latency LatencyModel, traceLevel int, sb SigningBacked, nn f3.NetworkName) *Network {
+	currentTime := time.Now()
```

Comment:
Replace with the zero `time.Time{}` for complete determinism, and no need for startTime.